features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/ecp/ecp_internal_alt.c

```diff
@@ -632,10 +633,9 @@ NU_STATIC int internal_run_eccop(const mbedtls_ecp_group *grp,
     crypto_ecc_prestart();
     CRPT->ECC_CTL = (grp->pbits << CRPT_ECC_CTL_CURVEM_Pos) | eccop | CRPT_ECC_CTL_FSEL_Msk | CRPT_ECC_CTL_START_Msk;
     ecc_done = crypto_ecc_wait();
-    
-    /* FIXME: Better error code for ECC accelerator error */
-    MBEDTLS_MPI_CHK(ecc_done ? 0 : -1);
-    
+
+    MBEDTLS_MPI_CHK(ecc_done ? 0 : MBEDTLS_ERR_SSL_HW_ACCEL_FAILED);
```

Comment:
`MBEDTLS_ERR_SSL_HW_ACCEL_FAILED` is for hardware accelerators that do TLS record processing, could we please return `MBEDTLS_ERR_PLATFORM_HW_ACCEL_FAILED` instead?

Reply:
@yanesca I fix it in https://github.com/ARMmbed/mbed-os/pull/8985/commits/406d480ad93d5a6aca9d3d7b36fff71004d27919.